flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_AvailableArrivalsPage.js

```diff
@@ -271,6 +256,21 @@ class CDUAvailableArrivalsPage {
                 down = true;
             }
             mcdu.setArrows(up, down, true, true);
+            mcdu.setTemplate([
+                ["ARRIVAL {small}TO{end} {green}" + airport.ident + "{end}"],
+                ["{sp}APPR", "STAR{sp}", "{sp}VIA"],
+                [selectedApproachCell + "[color]" + selectedApproachCellColor, selectedStarCell + "[color]" + selectedStarCellColor, "{sp}" + selectedViasCell + "[color]" + selectedViasCellColor],
+                [viasPageLabel, "TRANS{sp}"],
+                [viasPageLine, selectedTransitionCell + "[color]" + selectedTransitionCellColor],
```

Comment:
Did you mean to remove the old `[color]` everywhere? I see some are and some aren't e.g. here. I haven't noted the rest that remain.

Reply:
No, I only removed it in places where it was causing issues.